tests/test_utils.py

```diff
@@ -0,0 +1,66 @@
+import click.testing
+import pytest
+
+from dataclasses import dataclass
+from unittest import mock
+
+import moc_sprint_tools.cli
+
+
+@pytest.fixture
+def api(api_class, org):
+    mock_api_object = mock.Mock()
+    mock_api_object.organization = org
+
+    api_class.return_value = mock_api_object
+
+    yield mock_api_object
+
+
+@pytest.fixture
+def runner():
+    return click.testing.CliRunner()
+
+
+def test_shell_no_command(api, runner):
```

Comment:
This is also present in test_cli

Reply:
It doesn't seem to be. Are you looking at a stale checkout? Or am I just missing something obvious? I am looking at https://github.com/CCI-MOC/sprint-tools/blob/b5177152b1a94c4708bc97546a7eb27ba5796abc/tests/test_cli.py.

Reply:
The name is different but the following two lines are the same https://github.com/CCI-MOC/sprint-tools/blob/b5177152b1a94c4708bc97546a7eb27ba5796abc/tests/test_cli.py#L25